dcos/dcos-cluster-one-master.json

```diff
@@ -248,7 +248,7 @@
       "type": "string"
     }, 
     "linuxVersion": {
-      "defaultValue": "16.04.201605161", 
+      "defaultValue": "latest", 
```

Comment:
Just ran in the same same issue while playing with this template. The version is actually available but under another SKU. Try replacing `16.04.0-DAILY-LTS` by `16.04.0-LTS`.